// Copyright 2017 The Chromium OS Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

option optimize_for = LITE_RUNTIME;

package login_manager;

// ARC startup options sent from Chrome to session manager. This message is
// used in SessionManagerInterface.StartArcInterface method.
message StartArcInstanceRequest {
  // Account ID of the user to start ARC for. This must be the same as the
  // one given in StartSession.
  optional string account_id = 1;

  // Option to disable ACTION_BOOT_COMPLETED broadcast for 3rd party apps.
  optional bool skip_boot_completed_broadcast = 2;

  // Option to enable package manager service to scan /vendor/priv-app
  // directory.
  optional bool scan_vendor_priv_app = 3;

  // Next ID to use: 4
}
